src/selectize.js

```diff
@@ -158,6 +158,12 @@ $.extend(Selectize.prototype, {
 			$control_input.attr('placeholder', settings.placeholder);
 		}
 
+    // to have an identical rendering to a simple select (usefull for mobile device and do not open keyboard)
+    if (!self.settings.search) {
+      $control_input.attr('readonly', true);
```

Comment:
There is a problem with ReadOnly.

If the person clicks on the input and types something on the physical keyboard the input will remain empty, but selectize will filter the items and will work anyway.

To fix this just listen to the focus event and when the input receives focus you apply a blur. That way the input will never be focused and the person will never be able to type anything and we won't have any problems.

Reply:
A suggestion also to prevent the keyboard on mobile devices is to use $control_input.attr("inputmode", "none");

Some mobile browsers ignore the read_only property and show the keyboard anyway. But they respect input_mode.

The ideal is to have the 2 codes, read_only and input_mode none.

Reply:
thank you so much for your feedback, I fixed that !